File: guides/micronaut-turbo-native/java/src/main/java/example/micronaut/basic/OneController.java
package example.micronaut.basic;

import io.micronaut.http.MediaType;
import io.micronaut.http.annotation.Controller;
import io.micronaut.http.annotation.Get;
import example.micronaut.model.ViewModel;
import io.micronaut.views.View;

@Controller("/one")
class OneController {

    @View("one")
    @Get(produces = {MediaType.TEXT_HTML}, consumes = {MediaType.TEXT_HTML})
    ViewModel index() {
        return new ViewModel("How’d You Get Here?");
    }
}

File: guides/micronaut-turbo-native/python/src/example/micronaut/basic/one_controller.py
from micronaut.http import MediaType
from micronaut.http.annotation import Get, Produces
from micronaut.views import View

from ..model.view_model import ViewModel


@View("one")
@Get("/one")
@Produces(MediaType.TEXT_HTML)
def index() -> dict:
    return ViewModel("How’d You Get Here?").to_dict()
